node/pkg/driver/node.go

```diff
@@ -641,7 +643,12 @@ func (d *NodeService) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
-	err = d.NodeUtils.ExpandFilesystem(device, existingFormat)
+	mountPointToExpand := req.GetStagingTargetPath()
```

Comment:
`staging_target_path`  equal to `mount-point`?

Reply:
it is now.
alternatively, we could just ignore `staging_target_path` and always use `VolumePath`, whether it contains the publish or the stage path

Reply:
it will be the same for ext and xfs?

Reply:
yes, but it is not used for ext